android/player/src/main/java/com/intuit/playerui/android/lifecycle/PlayerViewModel.kt

```diff
@@ -160,7 +162,11 @@ public open class PlayerViewModel(flows: AsyncFlowIterator) : ViewModel(), Andro
             }
         }
 
-        release()
+        try {
+            release()
+        } catch (e: IllegalAccessError) {
+            Log.e("AndroidPlayer", e.message ?: "IllegalAccessError in AndroidPlayer")
+        }
```

Comment:
Do you think this might swallow real `IllegalAccessError`s? Can/should we be more targeted with what we're wrapping in try/catch, or maybe even not throw anything at all? i.e.:
```kotlin
    public fun release() {
        if (deferredPlayer.isCompleted) {
            player.release()
        }
    }
```